src/get-started/wordcount-example.md

```diff
@@ -190,26 +194,21 @@ The MinimalWordCount pipeline contains five transforms:
     {% github_sample /apache/beam/blob/master/sdks/python/apache_beam/examples/snippets/snippets.py tag:examples_wordcount_minimal_read
     %}```
 
-2.  A [ParDo]({{ site.baseurl }}/documentation/programming-guide/#pardo)
+2.  A FlatMapElements transform the PCollection of text lines.
```

Comment:
I think this first sentence is missing a word. maybe we could just remove this first line and start with the next line and add the transform name?  like:  "The FlatMapElements transform splits the lines in ..."